sdk/extensions/chronopost/karrio/mappers/chronopost/proxy.py

```diff
@@ -0,0 +1,45 @@
+from typing import Any
+from karrio.core.utils import (
+    Serializable,
+    Deserializable,
+    request as http,
+    XP,
+)
+from karrio.universal.mappers.rating_proxy import RatingMixinProxy
+from karrio.core.utils.helpers import exec_async
+from karrio.mappers.chronopost.settings import Settings
+from karrio.api.proxy import Proxy as BaseProxy
+
+
+class Proxy(RatingMixinProxy, BaseProxy):
+    settings: Settings
+
+    def _send_request(self, request: Serializable[Any], soapaction: str) -> str:
+        return http(
+            url=f"{self.settings.server_url}",
+            data=bytearray(request.serialize(), "utf-8"),
+            headers={
+                "Content-Type": "text/xml; charset=utf-8",
+                "soapaction": soapaction,
+            },
+            method="POST",
+        )
+
+    def get_rates(self, request: Serializable) -> Deserializable:
+        return super().get_rates(request)
```

Comment:
You will need to implement the API request here as per the QuickCost service documentation

```
https://ws.chronopost.fr/quickcost-cxf/QuickcostServiceWS/quickCost?
accountNumber=[accountNumber]&password=[password]&depCode=28500&arrCode=75001&weight=1&produ
ctCode=1&type=M
```